lms/templates/ccx/grading_policy.html

```diff
@@ -3,8 +3,8 @@
 <div id="warn-coach" class="wrapper-msg urgency-high warning">
   <div class="msg">
     <i class="msg-icon icon-warning-sign"></i>
-    <div class="msg-content">
-      <h3 class="title">${_("WARNING")}</h3>
+    <div class="msg-content" tabindex="0">
```

Comment:
@amir-qayyum-arbisoft, I'm not familiar with the bug this is supposed to fix and I don't know how to text CCX.  However, you generally do not want to make a non-focusable element, like a ```<div>``` focusable (tabindex=0) unless there is a JS listener on it and some behavior.  I don't see that here, so I'm wondering why this change was made.